contributors/devel/code-generation-outside-kubernetes.md

```diff
@@ -0,0 +1,54 @@
+# Code generation (from the experience of developing service-catalog)
```

Comment:
Super picky English observation.  You are writing a lot like me ... in passive voice ;)  I really try to write in active voice :)  Some cool online tools which can assist if you are interested.